src/ccs/multifolding.rs

```diff
@@ -160,33 +197,23 @@ impl<G: Group> Multifolding<G> {
       .map(|(a_i, b_i)| *a_i + b_i)
       .collect();
 
-    LCCCS {
-      matrix_mles: lcccs1.matrix_mles.clone(),
-      C,
-      ccs: lcccs1.ccs.clone(),
-      u,
-      x,
-      r_x: r_x_prime,
-      v,
-    }
+    self.lcccs.w_comm += cccs.w_comm.mul(rho);
```

Comment:
This code seems new, maybe a brief comment? Probably from other section somewhere

Reply:
Fixed in 52f6a62